### PR TITLE
Adding google 4 tag basic dimensions

### PR DESCRIPTION
**Pull request by pablo-code-org**
Google analytics migration is underway.
The measurement id `G-L9HT5MZ3HD` has been enabled through google tag manager.
Now we need to add the dimensions, events and cookies.
This is ONE of the places where we add such dimensions and the least aggressive and easier to test. 

For more information on the equivalence, look [here](https://support.google.com/analytics/answer/10271001?hl=en#zippy=%2Cin-this-article)

## Testing story

Testing is really fun.
1) Open [tag manager ](https://tagmanager.google.com/)
2) You can connect to a specific host (in this case localhost url). This will give you an output of the using the PREVIEW button
This loads the site 
image

From there we can go to the measurement tag added and go to the data layer sub tab
I can see the dimensions being loaded.
image

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/views/google_analytics.js.erb [hunk 1]
```diff
@@ -1,3 +1,14 @@
+window.dataLayer = window.dataLayer || [];
+function gtag(){dataLayer.push(arguments);}
+
+// Setup Google Analytics tag and dimensions
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/google_analytics.js.erb:4]: Can we expand the comments to differentiate between the GA4 vs. GA3 parts?

**Comment by breville** [comment]
> ## Warning!!

This warning can be removed.

> The measurement id `G-L9HT5MZ3HD` has been enabled through google tag manager.

Isn't that identifier used by GA4, while the other id `GTM-TZZBRK5` is being used for GTM?

**Comment by breville** [comment]
This covers pegasus, but we also need to do dashboard?  And it will have more dimensions available to be reported?

File: pegasus/sites.v3/code.org/views/google_analytics.js.erb (hunk 1 above)
Comment (pablo-code-org) [reply pegasus/sites.v3/code.org/views/google_analytics.js.erb:4]: Yes!

**Comment by pablo-code-org** [comment]
yes to all of the above

**Comment by pablo-code-org** [comment]
the thing is that all of these are spread around multiple files, I am trying to unified things into a single file

Also we need cookies and events to be migrated

**Pull request merged**